tests/Aspire.Azure.Npgsql.EntityFrameworkCore.PostgreSQL.Tests/Aspire.Azure.Npgsql.EntityFrameworkCore.PostgreSQL.Tests.csproj

```diff
@@ -7,6 +7,7 @@
   <ItemGroup>
     <Compile Include="$(RepoRoot)src\Aspire.Hosting.PostgreSQL\PostgresContainerImageTags.cs" />
     <Compile Include="..\Aspire.Npgsql.Tests\PostgreSQLContainerFixture.cs" />
+    <Compile Include="..\Aspire.Npgsql.EntityFrameworkCore.PostgreSQL.Tests\WorkaroundToReadProtectedField.cs" />
```

Comment:
There is more duplication we can get rid of:

* FakeTokenCredential
* ITestDbContext, TestDbContext
* CustomRetryExecutionStrategy
* CustomExecutionStrategy